beacon-chain/rpc/prysm/v1alpha1/validator/proposer.go

```diff
@@ -471,9 +471,15 @@ func (vs *Server) broadcastAndReceiveDataColumns(
 	slot primitives.Slot,
 ) error {
 	eg, _ := errgroup.WithContext(ctx)
-
 	dataColumnsWithholdCount := features.Get().DataColumnsWithholdCount
 
+	// Get the time corresponding to the start of the slot.
+	genesisTime := uint64(vs.TimeFetcher.GenesisTime().Unix())
+	slotStartTime, err := slots.ToTime(genesisTime, slot)
```

Comment:
As mentioned earlier , I think this can be done in the broadcast method itself, we do not need to provide it here.

Reply:
Addressed in the other comment.